### ISSUE TITLE
Design map interface

### ISSUE THREAD

**Issue by CourtneyLee333** [techmatters/terraso-product#99]
```[tasklist]
- [x] Wireframe approved
- [ ] Final Design approved
```

[LandPKS MUI Library](https://www.figma.com/file/7ZD5nE1IZKRHUIhrLkekoz/LandPKS-MUI-for-Figma-v5.11.1---Material-UI?node-id=8375%3A86793&t=e12gnjGPvVsDDbF9-1)
[LandPKS Wireframes and Prototype](https://www.figma.com/file/KMJbhjVyrrLFHj6jGcX9lN/LandPKS-Design-Prototype?t=hiT0127wb8Tey06i-1)

**Comment by CourtneyLee333** [comment]
@ltseng One of the pieces holding me up here is what to do when a user taps a site outside of the designated app terrirory (US vs. non-US). I have it in my notes that there are some details written out about this in a spec, but I can't find them. Where are we in the decision making process for this scenario, and the one in which a user wants to get soil and climate info and/or create a site outside the app territory?

**Comment by ltseng** [comment]
Oh man, this has changed multiple times over the course of this. I think the last we talked about this, we were going to let people create sites outside of their target location, but either warn them that the reports may not be accurate and/or block them from entering data. This all came out of the conversation around borders that Garo initiated a few spec review meetings back. Laura was going to check with Jeff about his opinions on out-of-region sites, but I never followed up with them.

**Comment by CourtneyLee333** [comment]
@ltseng Maybe it would be helpful to design something they can see to facilitate that conversation. I can work on that.

**Comment by CourtneyLee333** [comment]
Note that we are putting this edge case on hold for now, and will include it in a future issue.

**Comment by CourtneyLee333** [comment]
This is mostly done. I am waiting on the map discussion with engineers to decide whether the site card shows as a popover on the map, or in the bottom sheet.
